projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-submit/case-edit-submit.component.ts

```diff
@@ -158,33 +158,31 @@ export class CaseEditSubmitComponent implements OnInit, OnDestroy {
 
   public checkYourAnswerFieldsToDisplayExists(): boolean {
      /* istanbul ignore else */
-    if (!this.eventTrigger.show_summary) {
+    if (this.eventTrigger.show_summary || this.eventTrigger.show_summary === null) {
+      for (const page of this.wizard.pages) {
+        /* istanbul ignore else */
+       if (page.case_fields && this.isShown(page)) {
+         for (const field of page.case_fields) {
+            /* istanbul ignore else */
+           if (this.canShowFieldInCYA(field)) {
+             // at least one field needs showing
+             return true;
+           }
+         }
+       }
+     }
+    } else {
+      // found no fields to show in CYA summary page
       return false;
     }
-
-    for (const page of this.wizard.pages) {
-       /* istanbul ignore else */
-      if (this.isShown(page)) {
-        for (const field of page.case_fields) {
-           /* istanbul ignore else */
-          if (this.canShowFieldInCYA(field)) {
-            // at least one field needs showing
-            return true;
-          }
-        }
-      }
-    }
-
-    // found no fields to show in CYA summary page
-    return false;
   }
 
   public readOnlySummaryFieldsToDisplayExists(): boolean {
     return this.eventTrigger.case_fields.some(field => field.show_summary_content_option >= 0 );
   }
 
   public showEventNotes(): boolean {
-    return this.eventTrigger.show_event_notes === true;
+    return this.eventTrigger.show_event_notes || this.eventTrigger.show_event_notes === null;
```

Comment:
showEventNotes should _not_ be shown when not set. So this should be `!!this.eventTrigger.show_event_notes`